### DISCUSSION TITLE
Can homebrew provide a github template about package request?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [Homebrew/discussions#3891] [Writing Formulae/Casks]
Like <https://github.com/termux/termux-packages/blob/master/.github/ISSUE_TEMPLATE/02-package-request.yml> or <https://github.com/msys2/MINGW-packages/blob/master/.github/ISSUE_TEMPLATE/package_request.yml>?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew documents this. We won't add such templates because we only allow package requests as pull requests.
